salt/utils/thin.py

```diff
@@ -269,6 +269,8 @@ def gen_thin(cachedir, extra_mods='', overwrite=False, so_mods='',
     tempdir = None
     for py_ver, tops in six.iteritems(tops_py_version_mapping):
         for top in tops:
+            if not os.path.isabs(top):
+                continue
```

Comment:
Sorry, what was the reasoning behind this change?

Personally, I'm used to generate salt thin tarballs with local python packages, and it was working perfectly until this change. If it is not possible to remove this change, would it be possible to add an `absonly=False` arg to enable support for non absolute packages?